app/include/zmk/ble.h

```diff
@@ -13,6 +13,7 @@ int zmk_ble_clear_bonds();
 int zmk_ble_prof_next();
 int zmk_ble_prof_prev();
 int zmk_ble_prof_select(uint8_t index);
+int zmk_ble_reset_clear_all_bonds();
```

Comment:
I'd rather a naming like:

```suggestion
int zmk_ble_clear_all_profile_bonds();
```

And move this above the existing `zmk_ble_clear_bonds()`.